### DISCUSSION TITLE
Installation Error - "Python.h" not found

### DISCUSSION THREAD

**Discussion by AnuragKr** [mpi4py/mpi4py#287] [Q&A] [answered]
Hi,

I have been trying to install mpi4py but getting an error.
Command -- pip install mpi4py
System  Configuration -- DGX V100 Server

Error --

![Screenshot 2022-12-22 at 3 35 56 PM](https://user-images.githubusercontent.com/10074710/209118550-630057c8-c66a-4176-94bb-b13f64b4540d.png)

![Screenshot 2022-12-22 at 3 36 21 PM](https://user-images.githubusercontent.com/10074710/209118658-48d40493-c03d-4b03-8423-332c9c505270.png)

![Screenshot 2022-12-22 at 3 36 28 PM](https://user-images.githubusercontent.com/10074710/209118716-87e9f336-2acc-4f25-aea9-98ec5def04f7.png)

![Screenshot 2022-12-22 at 3 36 38 PM](https://user-images.githubusercontent.com/10074710/209118759-43ae3773-90f4-4e09-bd67-facd254777bc.png)

![Screenshot 2022-12-22 at 3 37 11 PM](https://user-images.githubusercontent.com/10074710/209119110-baa57fae-7805-4cc3-aca4-fe414c8fadfe.png)

![Screenshot 2022-12-22 at 3 37 23 PM](https://user-images.githubusercontent.com/10074710/209119143-45366445-90ab-4688-85e0-b17c6262ac8a.png)

Thanks

**Comment by dalcinl** [comment, marked as answer]
Your system is most likely missing Python development headers and libraries. You should be able to fix the problem by installing the Python development package with the following command:
```
sudo apt install python3.8-dev
```

**Reply by AnuragKr** [reply]
Thanks, the problem is fixed.
